sig-docs/annual-report-2021.md

```diff
@@ -4,79 +4,72 @@
-   - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-   - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+   - [1326 - Doc policies for third party content](https://git.k8s.io/enhancements/keps/sig-docs/1326-third-party-content-in-docs/README.md)
+
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Require more contributors to form a stable pool for SIG Docs
+   - Require guidelines on how we could ramp up folks nominated to the New Contributor Ambassador role specific to SIG Docs.
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+
+   - SIG Docs has a [dashboard](https://datastudio.google.com/u/0/reporting/fede2672-b2fd-402a-91d2-7473bdb10f04/page/567IC) available with site analytics. Some highlights include: 
+       - 2021 Pages views: 111,565,437
+       - Top pages for 2021 (excluding the home page): 
+           - https://kubernetes.io/docs/reference/kubectl/cheatsheet/
+           - https://kubernetes.io/docs/concepts/services-networking/service/
+           - https://kubernetes.io/docs/concepts/services-networking/ingress/
+   - PR velocity and open PR age is tracked in [Devstat](https://k8s.devstats.cncf.io/d/25/open-pr-age-by-repository-group?orgId=1&var-period=q&var-repogroup_name=SIG%20Docs&var-kind_name=All). We aim to have < 100 open PRs for the English localization, and will take steps as needed if we see the figure climbing much above that. For example, based on 12/31/2020 data, we have: Average number of opened PRs 368 with the median opened PRs age 5 days 21 hours 26 minutes 42 seconds
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - It's updated to the best of our knowledge.
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - We have a [well-documented guide](https://kubernetes.io/docs/contribute/#:~:text=Anyone%20can%20open%20an%20issue,the%20CNCF%20Contributor%20License%20Agreement.) that details how folks can get started and the various ways in which they can scale the contributor ladder.
+   - We actively assign good-first-issue labels to issues that we think early-lifecycle contributors could begin with. During KubeCons we also do introduction presentations and mentoring sessions to try and funnel contributors in to our SIG.
```

Comment:
have any links to those sessions?

Reply:
We didn't have any this past year. I combed through Youtube but only found Jim's, Zach's and Seokho's presentations at previous KubeCons. Would it be worthwhile listing them here? Jim's presentation is already listed below.

Reply:
There's also https://www.youtube.com/watch?v=pprMgmNzDcw and https://www.youtube.com/watch?v=vz_5EdhHrmI

Reply:
Added only the first one, since the second one is very generic and talks about docs in general.